blog/2021-07-30-supervisor_update.md

```diff
@@ -0,0 +1,72 @@
+---
+author: Joakim Sørensen
+authorURL: https://github.com/ludeeus
+authorImageURL: /img/profile/ludeeus.jpg
+authorTwitter: ludeeus
+title: Supervisor update
+---
+
+It's been a while since we posted about changes to the Supervisor. Here are some highlights from the past year and the future. This information is mainly for add-on developers, but there is little something for everyone in here. If you have not yet seen it, we have posted a [blog on the main site][main_blog] that you should read.
+
+## Snapshot -> Backup
+
+First up, as mentioned in the [blog on the main site][main_blog], we have started a transition away from the name "snapshot" that has been with us since the beginning of the Supervisor and are now moving to the more recognizable "backup".
+
+These changes are live now on the `dev` channel for the Supervisor, so you can start testing and adjusting your tools/add-ons to make sure they will still work when your users get this.
```

Comment:
I guess if we want to include it now this isn't really true.

```suggestion
```

Reply:
It's on dev.

Reply:
True, but it will probably not for all that long right?

```suggestion
These changes are live now on the `dev` channel (and will son be in the `beta` channel) for the Supervisor, so you can start testing and adjusting your tools/add-ons to make sure they will still work when your users get this.
```

Reply:
This post is aimed towards developers